docs/examples/promise/09_make_transfer_with_allowed_block_permissions_only/index.js

```diff
@@ -2,10 +2,11 @@
 /* eslint-disable @typescript-eslint/unbound-method */
 /* eslint-disable @typescript-eslint/no-var-requires */
 // Import the API, Keyring and some utility functions
-const { Api } = require('@cennznet/api');
-const testKeyring = require('@polkadot/keyring/testing');
-const { createType } = require('@cennznet/types');
 
+// const ExtrinsicEra = require( '@polkadot/types/primitive/Extrinsic/ExtrinsicEra');
```

Comment:
is this still needed?

Reply:
no.. I already removed this..in my clean up commit